src/OpenTK.GLControl/GLControl.cs

```diff
@@ -191,32 +212,37 @@ protected override void OnHandleCreated(EventArgs e)
                 implementation.WindowInfo.Dispose();
             }
 
-            if (design_mode)
-            {
-                implementation = new DummyGLControl();
-            }
-            else
+            try
             {
-                implementation = new GLControlFactory().CreateGLControl(format, this);
-            }
+                if (design_mode)
+                    implementation = new DummyGLControl();
+                else
+                    implementation = new GLControlFactory().CreateGLControl(format, this);
 
-            context = implementation.CreateContext(major, minor, flags);
-            MakeCurrent();
+                context = implementation.CreateContext(major, minor, flags);
```

Comment:
Check the indentation here. The short else only applies to one statement.

Reply:
Will do